setup/_menu.md

```diff
@@ -1,5 +1,52 @@
 # Setup
-1. [Dependencies & Downloads](dependencies.md)
-2. [Startup Guide](startup.md)
-3. [Additional PyCharm Setup](dependencies.md)
-4. [Checking Logs](logs.md)
+
+Before you start, you will need to have Python 3.7, Pycharm (community version is fine), and Docker installed on your system. Once you have these, you proceed with the steps below to setup the project.
+
+1. Go to https://repo.code.ubc.ca/
+You should see an option for "team_formation". 
+Select it.
+
+2. Resolve warning by adding your SSH key.
+On a Mac: You can check if there's an existing key and just add that. If not, generate one and then add it.
+
+3. Clone the repo onto your machine locally.
+On a Mac: Use "Clone with SSH" option, open a terminal and type:
+% git clone git@repo.code.ubc.ca:lt/team_formation.git
+On Windows: You might need to use "Clone with HTTPS" instead
+
+4. Open the project with Pycharm, TRUST the project, and set the python interpreter to version 3.7 (to match what's on your local)
+It will autorun: 
+% pip install -r requirements.txt
+Resolve issues if needed.
+Pycharm should have an interpreter (bottom right-hand corner) that shows a virtual environment for this project (e.g., "Python 3.7 (team_formation)").
+
+5. Get the .env file (from someone on the project already or get it from the project discord's server under "Notes/Resource") 
+Put in your project's root directory.
+
+6. Start docker.
+
+7. Run the project.
+On a Mac: In a terminal, run:
+% docker -compose up -d --build
```

Comment:
I don't think there's supposed to be a space between `docker` and `-compose`. It runs like `docker-compose up`